ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/zones/provision/cucushift-installer-rehearse-vsphere-ipi-zones-provision-chain.yaml

```diff
@@ -5,6 +5,7 @@ chain:
   - chain: ipi-install-vsphere
   - ref: enable-qe-catalogsource
   - chain: cucushift-installer-check
+  - ref: cucushift-installer-check-vsphere-cpms
```

Comment:
There is an existing vsphere tp job - `vsphere-ipi-proxy-tp-f28`, I think you can enable this step in that install workflow to cover signle-zone case?

Reply:
ok